runtime/src/lib.rs

```diff
@@ -553,6 +559,30 @@ impl pallet_preimage::Config for Runtime {
     type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+	pub const BasicDeposit: Balance = (1) as Balance * 2_000 * 10_000_000 + (258 as Balance) * 100 * 1_000_000;
```

Comment:
Why is the calculation like this? Is there some documentation that can be added?

Reply:
This is from the polkadot runtime.
https://github.com/paritytech/polkadot/blob/d74c60b6489f31f437edb1a19de0d0f6b8ad432e/runtime/polkadot/src/lib.rs#L620